### ISSUE TITLE
Push notification issue

### ISSUE THREAD

**Issue by qzaqza1818** [QuickBlox/quickblox-android-sdk#133]
I was able to subscribe successfully by following your tutorial, the evidence is it appeared in the admin console, push notification, Subscriptions. Afterwards, I try to send notification from the admin console to test whether it works or not.

I set testing for environment, android for channels and a sentence for the message. There is a success notification appear in the admin console but I don't saw any notification appear in my device. 

Besides that, the device which had been subscribed was disappeared right after the notification pushed successfully.

In summary, there are two problems I faced. First is I didn't receive notification in my device even though there is a success notification appeared in admin console. Second is the device which had been registered or subscribed successfully, not to mention that it appear in the Subscriptions page, was gone/disappeared right after the notification being pushed.

**Comment by soulfly** [comment]
Can you check the following:

1) Check Push Notifications Queue tab in Admin panel, there should be a log of your push. It will help you to understand what's going on with your devices.

2) Maybe you got 'MismatchSenderId' error there. In this case, our backend can remove the tokens because they are invalid. Can you check the following:
" you use SenderID in Android app and Google API Key in QuickBlox admin panel from the same Google app"
